test/check-session

```diff
@@ -54,13 +56,13 @@ class TestSession(MachineCase):
         b.click('#login-button')
         b.expect_reload()
         b.wait_page("server")
```

Comment:
The question here is: what state do we assert after showing login screen again.
If we consider logout to be asynchronous, the session might not be fully terminated, so a wait_session() is ok.